config/rapidez/checkout-theme.php

```diff
@@ -23,6 +23,7 @@
         ],
     ],
     'checkout' => [
+        'show_crosssells' => true,
```

Comment:
It's the cart, not the checkout. We do want/need this configurable?

Reply:
I guess you can "disable" it by just not configuring any cross-sells on any products so it's not really necessary